Cargo.toml

```diff
@@ -59,6 +59,7 @@ url = "1.0"
 default = [
     "runtime",
     "stream",
+    "log"
```

Comment:
This should be removed (and the `log = ["tracing/log"]` as well).

Reply:
should be resolved